tests/utils/index.js

```diff
@@ -1218,6 +1219,20 @@ const updatePermissions = async (roles, addPermissions, removePermissions, ignor
   await updateSettings({ permissions: settings.permissions }, ignoreReload);
 };
 
+const saveFeedbackDocs = async (filename, existingFeedbackDocIds) => {
+  const feedBackDocs = await chtDbUtils.getFeedbackDocs();
+  const newFeedbackDocs = feedBackDocs.filter(doc => !existingFeedbackDocIds.includes(doc._id));
+  if (!newFeedbackDocs.length) {
+    return false;
+  }
+
+  const fileName = `feedbackDocs-${filename}.json`;
```

Comment:
This is hard to read, partially because of the difference in case `fileName` vs `filename`. The parameter should change because it's not actually the filename, it's just part of the filename.

My suggestion is to take the `test` as the parameter and then do all the formatting here in one place, eg:

```
const filename = `feedbackDocs-${test.parent}-${test.title}.json`.replace(/\s/g, '-');
```